### ISSUE TITLE
Project creation, new group => no label on fields

### ISSUE THREAD

**Issue by abretaud** [genouest/genouestaccountmanager#508]
When I try to validate a project, I click on the + button to create a group => there's no label next to the fields, so it's hard to understand what is written

bonus: the + button is not super explicit, it would be nice to have a tooltip at least

**Comment by mboudet** [comment]
You mean, in the group creation modal?

![image](https://github.com/user-attachments/assets/6ca15b53-6d71-4636-9755-52c665ec68e8)

**Comment by abretaud** [comment]
yep, when it's prefilled with values from the project, it's not super understandable (forced to delete stuff to understand)
